src/pages/Dashboard/Apps/MyApps/index.jsx

```diff
@@ -101,9 +104,16 @@ export default class Apps extends Component {
       <Layout className={styles.layout} noSubMenu={true}>
         <Loading className={styles.page} isLoading={pageLoading}>
           {this.renderHeader()}
-          <div className={styles.cards}>
-            {apps.map(item => <Card key={item.app_id} t={t} data={item} />)}
-          </div>
+          <InfiniteScroll
+            pageStart={appStore.currentPage}
```

Comment:
Why not extract variables from appStore?

Reply:
ok